lacework-agent/templates/configmap.yaml

```diff
@@ -37,10 +37,18 @@ data:
        {{- end }}
      {{- end }}
      {{- with .Values.laceworkConfig.codeaware }}
-       {{- if .enable }}
+       {{- if or (eq (quote .enable) "true") (eq (quote .enable) "all") }}
         "codeaware": {
           "enable": "all"
         },
+       {{- else if (eq (quote .enable) "experimental") }}
```

Comment:
May be we can combine these statements to the equivalent of
{{- else if (eq (quote .enable) "experimental") } or {(eq (quote .enable) "false")}}
    "codeaware": {
          "enable":  (quote .enable)
        },

Reply:
Good idea, filed https://github.com/lacework/agent/pull/2907 to do this